src/components/MainHeader.tsx

```
@@ -0,0 +1,297 @@
+              {isAuthenticated && (
+                <li className="pt-2 border-t border-neutral-100">
+                  <UnifiedAuthComponent />
+                </li>
+              )}
+            </ul>
+          </nav>
+        </div>
+      </div>
+
+      {/* Keyboard shortcuts */}
+      <KeyboardShortcut
```

ESLint

[error] 'KeyboardShortcut' is not defined.

(react/jsx-no-undef)